platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformComponentBuilder.java

```diff
@@ -534,4 +536,9 @@ public ConsensusEngine buildConsensusEngine() {
         }
         return consensusEngine;
     }
+
+    @NonNull
+    public SignedStateSentinel buildSignedStateSentinel() {
+        return new DefaultSignedStateSentinel(blocks.platformContext());
+    }
```

Comment:
tab complete didn't work from blank.  I wonder if its because I don't have a history for today of developing pieces.   Once I put the method name I was able to tab complete.